web/src/pages/Route/List.tsx

```diff
@@ -494,7 +494,7 @@ const Page: React.FC = () => {
           <p>{formatMessage({ id: 'page.route.instructions' })}:</p>
           <p>
             <a
-              href="https://github.com/apache/apisix-dashboard/blob/master/docs/IMPORT_OPENAPI_USER_GUIDE.md"
+              href="https://github.com/apache/apisix-dashboard/blob/master/docs/en/latest/IMPORT_OPENAPI_USER_GUIDE.md"
```

Comment:
Hi, we could use the new doc site here:

https://apisix.apache.org/docs/dashboard/IMPORT_OPENAPI_USER_GUIDE